cuckoo/apps/rooter.py

```diff
@@ -348,48 +348,52 @@ def cuckoo_rooter(socket_path, group, ifconfig, service, iptables, ip):
     s.ip = ip
 
     while True:
-        command, addr = server.recvfrom(4096)
-
         try:
-            obj = json.loads(command)
-        except:
-            log.info("Received invalid request: %r", command)
-            continue
-
-        command = obj.get("command")
-        args = obj.get("args", [])
-        kwargs = obj.get("kwargs", {})
-
-        if not isinstance(command, basestring) or command not in handlers:
-            log.info("Received incorrect command: %r", command)
-            continue
-
-        if not isinstance(args, (tuple, list)):
-            log.info("Invalid arguments type: %r", args)
-            continue
-
-        if not isinstance(kwargs, dict):
-            log.info("Invalid keyword arguments: %r", kwargs)
-            continue
-
-        for arg in args + kwargs.keys() + kwargs.values():
-            if not isinstance(arg, basestring):
-                log.info("Invalid argument detected: %r", arg)
-                break
-        else:
-            log.debug(
-                "Processing command: %s %s %s", command,
-                " ".join(args),
-                " ".join("%s=%s" % (k, v) for k, v in kwargs.items())
-            )
-
-            output = e = None
+            command, addr = server.recvfrom(4096)
```

Comment:
Please un-indent this entire code block to restore its initial indentation ;-)
Or actually, I'm not sure yet about the whole `EINTR` thing yet.